ci-operator/jobs/openshift/dpu-operator/openshift-dpu-operator-main-presubmits.yaml

```diff
@@ -62,7 +62,7 @@ presubmits:
       - name: result-aggregator
```

Comment:
Shouldn't "always_run" be false here as well?

Reply:
Looking above in this section.

Reply:
why would result aggregator be always run in this case? And this presubmit is generated by `make jobs`